tests/Propel/Tests/Generator/Model/ColumnTest.php

```diff
@@ -95,7 +95,7 @@ public function testSetupObjectWithPlatformOnly()
             ->method('getType')
             ->will($this->returnValue('VARCHAR'));
 
-        $column = new Column();
+        $column = new Column('');
```

Comment:
Do we need all these changes?
Couldnt null internally set the empty string in both cases?

Reply:
An empty string here is good since it is a test. But I'm not sure that empty string as a default name for the real column is a good choice.